### ISSUE TITLE
Behaviours marked as failed in R# runner, even though they passed.

### ISSUE THREAD

**Issue by NameOfTheDragon** [machine/machine.specifications.runner.resharper#16]
My behaviors are marked as failed in R# runner, even though I have a 'green bar' and the fail count is reported as 0. They also pass in the console runner, so it's pretty sure this is a R# runner issue.
![bad behaviour in resharper](https://cloud.githubusercontent.com/assets/1268490/7203259/44b48770-e511-11e4-932b-641167e9c3f7.PNG)
![good behaviour in console](https://cloud.githubusercontent.com/assets/1268490/7203282/70dd931e-e511-11e4-8733-de9c122780be.PNG)

**Comment by danielmarbach** [comment]
Version?

**Comment by NameOfTheDragon** [comment]
Apologies, I should have known to provide thatinformation.
MSpec 0.9.1
ReSharper 9.0 update 1
ReSharper Runner 1.0.0

I’ll try to make a repro solution but I have a dozen things I need to get done in the next hour or two.
--T

**Comment by NameOfTheDragon** [comment]
Weirdness ensues. I tried to make a repro solution and couldn’t. Then I went back to my original problem solution, and I now can’t reproduce the issue there either. It was definitely a real thing though as you can see from the screen shots I attached. It has got to be some weird caching thing in ReSharper, probably.

Given that I can no longer reproduce, do you want to close the issue?
--Tim

**Comment by danielmarbach** [comment]
Close it and reopen if it turns up again

**Comment by NameOfTheDragon** [comment]
I couldn't reproduce 'at will' but I did notice a further oddity that is work mentioning. On several occasions I completed a test run with a 'green bar'. The run was completely finished, but by expanding/collapsing a behaviour node, I could get the node to flip from pass to fail.

Nodes changing state after-the-fact seems like a presentation issue to me, since I assume once the runner has reported its results they are immutable.

Anyway I'm sorry I can't tie it down any further. I am trying out NCrunch at the moment so the R# runner is on hold for the time being. I'll close this issue for now and someone else can re-open it if they see the same symptoms.
